src/Facebook/InstantArticles/Client/Client.php

```diff
@@ -81,20 +81,26 @@ public static function create($appID, $appSecret, $accessToken, $pageID, $develo
     /**
      * Import an article into your Instant Articles library.
      *
-     * @param InstantArticle $article The article to import
+     * @param string|InstantArticle $article The article to import, either as a string or an InstantArticle object.
      * @param bool|false $takeLive Specifies if this article should be taken live or not. Optional. Default: false.
      */
     public function importArticle($article, $takeLive = false)
```

Comment:
@simonengelhardt @alucic

Since `$article` isn't type-hinted, this shouldn't even be a breaking change, right?